src/3rdparty/walkontable/src/table.js

```diff
@@ -787,7 +786,7 @@ class Table {
   }
 
   isRowAfterRenderedRows(row) {
-    return this.rowFilter && (row > this.getLastRenderedRow());
+    return this.rowFilter && (this.rowFilter.sourceToRendered(row) > this.getLastRenderedRow());
```

Comment:
This was recently changed by @warpech PR. Please communicate before changing it back and forth @jansiegel @warpech @pnowak  

https://github.com/handsontable/handsontable/pull/6079/files#diff-0a25aaaa72d09482c0875ad092179097L567

Reply:
Yeah, my change didn't work properly, it's reverted in the next commit.